src/current/_includes/v24.2/sql/row-level-ttl-prefer-ttl-expiration-expressions.md

```diff
@@ -1,4 +1,5 @@
-Starting with CockroachDB v22.2, we recommend that most users use `ttl_expiration_expression` over `ttl_expire_after` for the following reasons:
+Most users should use `ttl_expiration_expression` instead of `ttl_expire_after` for the following reasons:
 
+- If you add `ttl_expire_after` to an existing table, it **will cause a full table rewrite, which can affect performance**. Specifically, it will result in a [schema change]({% link {{ page.version.version }}/online-schema-changes.md %}) that (1) creates a new [hidden column]({% link {{page.version.version}}/show-create.md%}#show-the-create-table-statement-for-a-table-with-a-hidden-column) `crdb_internal_expiration` for all rows, and (2) backfills the value of the that new column to `now()` + `ttl_expire_after`.
+- If you use `ttl_expire_after`, you can't use it with an existing [`TIMESTAMPTZ`]({% link {{ page.version.version }}/timestamp.md %}) column.
```

Comment:
Alternatively, would this be more concise?

```suggestion
You cannot use `ttl_expire_after` with an existing [`TIMESTAMPTZ`]({% link {{ page.version.version }}/timestamp.md %}) column.
```

Reply:
✅